core/classes/Adapter.js

```diff
@@ -426,10 +426,20 @@ module.exports = class Adapter {
         if (Array.isArray(schema.required) && schema.required.includes(key)) {
           attribute.notNullable()
         }
+        if (property.hasOwnProperty("default")) {
+          attribute.defaultTo(property.default)
+        }
         if (property.unique) {
           table.unique(key)
         }
       })
+      if (schema.uniqueKeys && typeof schema.uniqueKeys === "object") {
+        _.forEach(schema.uniqueKeys, (value) => {
+          if (Array.isArray(value)) {
```

Comment:
And what if not? You simple do nothing? And let the other programmers guess, why doesn't their uniqueKeys work? It's always better to throw an exception in such cases. But in this case it's better to let it into the knex – it work swell even if the arg is not an array